docker/images/kuma_base/Dockerfile

```diff
@@ -12,15 +12,10 @@ RUN set -x \
     && apt-get install -y --no-install-recommends \
     curl \
     dirmngr \
+    # Needed for pytz to be able to install
     libsasl2-modules \
     gettext \
-    build-essential \
-    libxml2-dev \
-    libxslt1-dev \
-    libffi-dev \
-    libmagic-dev \
-    default-libmysqlclient-dev \
-    default-mysql-client
+    build-essential
```

Comment:
Why should we do that? We're using `psycopg2-binary` in `pyproject.toml` and clearly the container works after the existing `Dockerfiler` has been built which includes the `poetry install`. 
Is `postgresql-client` making this better or faster?

Reply:
@peterbe and I discussed this earlier today over a video call, but I'll summarize the key points here. The motivation behind my request was for the convenience of running the `psql` client from any of the Kubernetes pods that run the Kuma image. In the end, we decided to leave it out because for those times when I need to run `psql` within Kubernetes, I can `kubectl exec` into the "admin" pod, on which I've already installed the `psql` client.